.eslintrc.js

```diff
@@ -16,6 +18,12 @@ module.exports = {
       rules: {
         'header/header': ['off']
       }
+    },
+    {
+      files: ['test/**/*.js'],
```

Comment:
we have a .eslintrc.js in test. it might be more appropriate to put this in that config

Reply:
Fixed.